src/main/java/core/basesyntax/dao/CarJdbcDaoImpl.java

```diff
@@ -0,0 +1,187 @@
+                + "JOIN drivers d on d.driver_id = cd.driver_id "
+                + "JOIN cars c on c.car_id = cd.car_id "
+                + "JOIN manufacturers m on m.manufacturer_id = c.manufacturer_id "
+                + "WHERE cd.driver_id = ? AND c.deleted = FALSE AND d.deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement driversByCarStatement = connection.prepareStatement(query)) {
+            driversByCarStatement.setLong(1, driverId);
+            ResultSet result = driversByCarStatement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (result.next()) {
+                cars.add(getCar(result, connection));
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get car by driver id " + driverId, e);
+        }
+    }
+
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars(model, manufacturer_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createdCar = connection
+                        .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            createdCar.setString(1, car.getModel());
+            createdCar.setLong(2, car.getManufacturer().getId());
+            createdCar.executeUpdate();
+            ResultSet resultSet = createdCar.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t create car " + car, e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT * FROM cars c "
+                + "JOIN manufacturers m on m.manufacturer_id = c.manufacturer_id "
+                + "WHERE car_id = ? AND c.deleted = FALSE";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement gettingCar = connection.prepareStatement(query)) {
+            gettingCar.setLong(1, id);
+            ResultSet result = gettingCar.executeQuery();
+            if (result.next()) {
+                car = getCar(result, connection);
+            }
+            return Optional.ofNullable(car);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t add car by id " + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String query = "SELECT * FROM cars c "
+                + "JOIN manufacturers m on m.manufacturer_id = c.manufacturer_id "
+                + "WHERE c.deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement gettingAll = connection.prepareStatement(query)) {
```

Comment:
```suggestion
                PreparedStatement getAllCarsStatement = connection.prepareStatement(query)) {
```
same with others. for me your statements names look like some methods names